src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackService.java

```diff
@@ -61,4 +83,38 @@ public CreateCaseCallbackService(
             .filter(item -> !Strings.isNullOrEmpty(item.getTransformationUrl()))
             .getOrElse(Validation.invalid("Transformation URL is not configured"));
     }
+
+    private Validation<Seq<String>, Map<String, Object>> createNewCase(
+        ExceptionRecord exceptionRecord,
+        ServiceConfigItem configItem,
+        long caseId
+    ) {
+        try {
+            log.info(
+                "Start creating exception record for service {} and exception record {}",
+                configItem.getService(),
+                caseId
+            );
+
+            transformationClient.transformExceptionRecord(
+                configItem.getTransformationUrl(),
+                exceptionRecord,
+                "s2s token"
+            );
+
+            return Validation.valid(ImmutableMap.of("caseReference", UUID.randomUUID()));
+        } catch (InvalidCaseDataException exception) {
```

Comment:
At this point 422 and 400 responses are indistinguishable from each other. Whatever handles this exception needs to have that information, because a 422 response will result in us returning a 200 response to CCD, with errors and warnings (presented to caseworker), while 400 means there's an error at service level (orchestrator can't properly talk to the transformation service) and it's going to be a 500 for CCD.